src/deephaven/plugin/matplotlib/__init__.py

```diff
@@ -1,10 +1,13 @@
 from deephaven.plugin import Registration
+from deephaven.table_listener import listen
```

Comment:
You'll need to declare a dependency on deephaven if you want to do this, but unfortunately we don't publish deephaven yet.

Reply:
If we don't publish deephaven... then what can I do?

Reply:
At the very least, we need to have a note in the README and update our setup install_requires.

Reply:
I'll update the `install_requires`